typescript/ws-protocol/src/types.ts

```diff
@@ -78,12 +80,22 @@ export type MessageData = {
   timestamp: bigint;
   data: DataView;
 };
+export type TimeData = {
```

Comment:
name doesn't match opcode – was there a naming conflict?

Reply:
nope, will update it